app/src/main/java/com/example/calculator/mvp/view/CalculatorView.java

```diff
@@ -17,4 +21,26 @@ public CalculatorView(Activity activity, ActivityMainBinding binding) {
     public void showOperationValue(String operation) {
         binding.textViewOperations.setText(operation);
     }
+
+    @Override
+    public void showResultValue(String result) {
+        binding.textViewResult.setText(result);
+    }
+
+    @Override
+    public void showWrongOperator() {
+        showResultValue(WRONG_OPERATOR_ERROR);
```

Comment:
do not use constants here. you are in the view, so you can access string resources

Reply:
Solved!